scripts/restore-sanitised-database.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+SANITISED_DB_PATH="Sanitised-database.sql"
+S3_BUCKET=""
+LOCAL_DB_PASSWORD="password"
+LOCAL_DB_USER="postgres"
+LOCAL_DB_NAME="data_store"
+DB_IMAGE="postgres:16.2"
+
+# Function to list and select an AWS Vault profile
+select_aws_vault_profile() {
+    local prompt_message=$1
+    local profile_variable=$2
+    local allowed_profile=$3
+
+    while true; do
+        echo "$prompt_message"
+        PS3='Please select a profile (or Ctrl-C to quit): '
+        IFS=$'\n' read -r -d '' -a profiles <<< "$(aws-vault list --profiles)"
+        select profile in "${profiles[@]}"; do
+            if [[ -n "$profile" ]]; then
+                ACCOUNT=$(aws-vault exec "$profile" -- aws sts get-caller-identity --query 'Account' --output text)
+                if [[ "$allowed_profile" == "prod" && ! "$ACCOUNT" =~ ^233 ]]; then
+                    echo "You must select a production profile. Please try again."
+                elif [[ "$allowed_profile" == "non_prod" && "$ACCOUNT" =~ ^233 ]]; then
+                    echo "Production profile is not allowed. Please select a non-production profile."
+                else
+                    export "$profile_variable"="$profile"
+                    return
```

Comment:
Yeah let's remove some of this stuff - this script should never be able to restore to prod - that would be Big-B Bad.